zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/ClusterManager.java

```diff
@@ -470,6 +487,7 @@ public void deleteClusterMeta(ClusterMetaType type, String key) {
       .register(HashMap.class)
       .register(ClusterMetaEntity.class)
       .register(Date.class)
```

Comment:
Date can be removed ? since you use the new java date api

Reply:
Ok, But if need to transfer objects of type Date in the cluster later, Need to add this code back.